tests/ballerina-unit-test/src/test/resources/test-src/expressions/builtinoperations/clone-operation.bal

```diff
@@ -0,0 +1,179 @@
+type Employee record {
```

Comment:
Shall we add the license header to this file?

Reply:
Fixed